### DISCUSSION TITLE
Adding Pretrained object detection models like DETR

### DISCUSSION THREAD

**Discussion by pranavjadhav001** [keras-team/keras-cv#616] [General]
Are we thinking of supporting object detection pretrained models? If yes, DETR is anchor free model, with pretrained weights available. We can provide some functionality right now like inference of pretrained models and extend to training functionality in future. Do we have anything similar in pipeline?

**Comment by innat** [comment]
@pranavjadhav001 It's a strong model. You can open a ticket and if the keras team agrees to have you can send PR. Here is one tensorflow reference of this model, check https://github.com/Visual-Behavior/detr-tensorflow

**Comment by bhack** [comment]
https://github.com/tensorflow/models/tree/master/official/projects/detr

**Comment by pranavjadhav001** [comment]
https://github.com/pranavjadhav001/detr_keras I've also implemented a tf version of DETR

**Comment by jbischof** [comment]
@pranavjadhav001 we've not firmed up our object detection API and have RetinaNet and YOLOv8. I think we're ready for more! Would you still be interested in porting DETR?
